GitHub-Test-3/GC-gitTest3.md

```diff
@@ -0,0 +1,8 @@
+# GitTest 3 
+test try to type some in here
```

Comment:
@gzc5211 Okay! You've written a little markdown here. That `#` created a top-level header. 
What I'm writing here is an example of a **line comment**, which you can write on a commit to a repo on GitHub. You'll be writing one of these on a commit for the last part of the Github test.